packages/documentation/plugins/github-api-pages/pages/index.js

```diff
@@ -0,0 +1 @@
+exports.workPractices = require('./work-practices.json');
```

Comment:
I think we could just have one JSON file, until we get to the point where that's too painful (which I think is unlikely).